### PR TITLE
First attempt to adding share on social buttons

### PR DESCRIPTION
**Pull request by tassoman**
Ciao! :grin: 
This is my attempt to add two share buttons. 

Please note that this code doesn't contain all strings translated. Only italian and english.
More, I have no clue how to catch current url while being in html code. 
Maybe must be overwritten on document ready by jquery using window.location?

Please have a look and file your suggestion or make modifications accordingly.
Keep in touch :beers:

### CODE REVIEW

**Comment by nlehuby** [comment]
Hi !

Nice job :)

Why not using images instead of the strings "Share to whatever" ? It will spare us the translation job for this new feature ;)

To get the current url on the sharing link, I rather suggest a little onclick js function to add the map.getCenter() lat & lon to the href url.

If you want to add Facebook, please consider adding Diaspora* too : https://sharetodiaspora.github.io/?title=Hey let's meet there&url=http://openbeermap.github.io

**Comment by tassoman** [comment]
I've relied on current _document.URL_ to build the sharing url :)

File: index.html [hunk 1]
```diff
@@ -58,9 +58,16 @@
...
+                    <li><a class="modal-about-toggle" href="#" data-toggle="collapse" data-target=".navbar-collapse.in" data-l10n-id="nav_link_about"><i class="fa fa-fw fa-lg fa-question-circle"></i> About</a></li>
+                    <li><a href="https://www.openstreetmap.org/user/OpenBeerMapContributor/diary" target='_blank' data-l10n-id="nav_link_news"><i class="fa fa-fw fa-lg fa-rss"></i> News</a></li>
+                    <li class="navbar-pulled">
+                      <ul class="nav navbar-nav">
+                          <li><a href="#" onclick="social_share('diaspora');"><i class="fa fa-fw fa-lg fa-asterisk"></i></a></li>
+                          <li><a href="#" onclick="social_share('twitter');"><i class="fa fa-fw fa-lg fa-twitter"></i></a></li>
+                          <li><a href="#" onclick="social_share('facebook');"><i class="fa fa-fw fa-lg fa-facebook"></i></a></li>
+                          <li><a href="https://flattr.com/submit/auto?user_id=nlehuby&url=http%3A%2F%2Fopenbeermap.github.io" target="_blank"><img src="assets/img/3rdparties/flattr-badge-large.png" alt="Flattr this" title="Flattr this" border="0"></a></li>
```
Comment (OpenBeerMap) [inline index.html]: please add target to open the sharing links in a new tab

File: index.html [hunk 2]
```diff
@@ -58,9 +58,16 @@
...
-                    <li class="navbar-pulled"><a href="https://flattr.com/submit/auto?user_id=nlehuby&url=http%3A%2F%2Fopenbeermap.github.io" target="_blank"><img src="assets/img/3rdparties/flattr-badge-large.png" alt="Flattr this" title="Flattr this" border="0"></a></li>
+                    <li><a class="modal-about-toggle" href="#" data-toggle="collapse" data-target=".navbar-collapse.in" data-l10n-id="nav_link_about"><i class="fa fa-fw fa-lg fa-question-circle"></i> About</a></li>
+                    <li><a href="https://www.openstreetmap.org/user/OpenBeerMapContributor/diary" target='_blank' data-l10n-id="nav_link_news"><i class="fa fa-fw fa-lg fa-rss"></i> News</a></li>
+                    <li class="navbar-pulled">
+                      <ul class="nav navbar-nav">
+                          <li><a href="#" onclick="social_share('diaspora');"><i class="fa fa-fw fa-lg fa-asterisk"></i></a></li>
+                          <li><a href="#" onclick="social_share('twitter');"><i class="fa fa-fw fa-lg fa-twitter"></i></a></li>
+                          <li><a href="#" onclick="social_share('facebook');"><i class="fa fa-fw fa-lg fa-facebook"></i></a></li>
```
Comment (nlehuby) [inline index.html]: please add target to open the sharing links in a new tab

**Comment by nlehuby** [comment]
Too bad : the current location is not exported to twitter (everything after the # is ignored) :disappointed:

**Comment by tassoman** [comment]
ooOoops! Let me edit further! :wink:

**Comment by nlehuby** [comment]
hi @tassoman !
any update on this PR ?

**Comment by tassoman** [comment]
hello @nlehuby , [I'm getting the url by `document.URL` js object](https://github.com/tassoman/OpenBeerMap.github.io/commit/a36e028e818c05d7b909e47f1f29c1e3f3a075c6#diff-41cc00af96ba1f255920c5a8a682598bR61). Is it correct? Looks like the others social networks are working flawless. Maybe it's Twitter cutting away uris?

**Pull request merged**